test/index.ts

```diff
@@ -1793,7 +1797,39 @@ describe('BigQuery', () => {
       });
     });
 
-    it('should call job#getQueryResults', done => {
+    it('should call table#getRows', done => {
+      const fakeJob = new EventEmitter();
+      // tslint:disable-next-line: no-any
```

Comment:
I recently realized that codecov will aggregate together coverage results from multiple reporters. Along with adding tests (which is awesome) it might be worth changing the `system-test` script in `package.json` to the following:

```
 "system-test": "nyc mocha build/system-test --timeout 600000",
```

Any integration tests that exercise your code path will then also be reflected in codecov 😄

Reply:
Got it. Thanks!